### PR TITLE
add chimba network

### PR DESCRIPTION
**Pull request by camiloss1**

### CODE REVIEW

File: chimba/chain.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "chimba",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://chimba.ooo/",
+  "pretty_name": "Chimba",
+  "chain_id": "chimba",
+  "bech32_prefix": "chimba",
```
Comment (JeremyParish69) [inline chimba/chain.json:9]: Please correct the SLIP-173 registration at https://github.com/satoshilabs/slips/blob/master/slip-0173.md . It has 'cmba' as the bech32 prefix, when in fact the prefix is 'chimba'.

File: chimba/chain.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "chimba",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://chimba.ooo/",
+  "pretty_name": "Chimba",
+  "chain_id": "chimba",
```
Comment (JeremyParish69) [inline chimba/chain.json:8]: Looks like the chain-id might be chimbanode-1, as per what I see on the explorer.

File: chimba/chain.json (hunk 1 above)
Comment (camiloss1) [reply chimba/chain.json:9]: the change in the file for slip-073 has already been applied

File: chimba/chain.json (hunk 2 above)
Comment (camiloss1) [reply chimba/chain.json:8]: chimbanode-1 is the name for moniker but the chain-id is chimba

**Pull request merged**